slam-libraries/viam-orb-slam3/orbslam_server_v1_test.cc

```diff
@@ -133,7 +133,7 @@ BOOST_AUTO_TEST_CASE(ReadTimeFromFilename) {
 }
 
 BOOST_AUTO_TEST_CASE(FindFrameIndex_Closest_no_files) {
-    const string configTimeString = "2022-01-01T01_00_00.0000";
+    const string configTimeString = "2022-01-01T01:00:00.0000";
```

Comment:
Why doesn't this use Z?

Reply:
good catch, forgot to include the Z when updating the tests